README.rst

```diff
@@ -10,6 +12,20 @@ To get ``dynsight``, you can install it with pip::
     $ pip install dynsight
 
 
+DEV installation
+============
+
+If you want to setup a development version of ``dynsight`` you can follow this simple steps:
+    1. Fork the ``dynsight`` repository on your github account by clicking the ``Fork`` button in the github page.
+    2. Clone your forked repository by using ``git clone <YOUR_LINK>`` (fill properly).
+    3. Create a new virual environment or ``conda`` or ``mamba`` (reccomended) using Python 3.10 version.
+    4. Install ``just`` (https://github.com/casey/just).
+    5. Activate the virtual environment created and run ``just dev`` within the cloned repository.
+
```

Comment:
Maybe change the above to something like (my rationale is that you do not need to tell people how to clone a repository):

```
#. Install `just`_.
#. Fork the ``dynsight`` repository
#. In a new virtual environment run using Python 3.10::

    $ just dev

#. Run code checks::

    $ just check
  


.. _`just`: https://github.com/casey/just
```

Reply:
Check formatting.